### ISSUE TITLE
error

### ISSUE THREAD

**Issue by virla01** [kriollo/login#3]
Hola estaba probando tu arreglo y lo unico que llegue fue a logearme, despues no me deja ver nada

`Call to a member function fetchAll() on boolean
--
(1/1) FatalThrowableErrorCall to a member function fetchAll() on boolean
--
en Database.php (línea 272)
de Database->query_select('SELECT * FROM empresa WHERE 1 = 1 ;')en Database.php (línea 262)
de Database->select('*', 'empresa')en Empresa.php (línea 46)
de Empresa->get()en Controllers.php (línea 130)
de Controllers->__construct(object(Router), array('users_logged' => true))en administracionController.php (línea 29)
de administracionController->__construct(object(Router))en Router.php (línea 241)
de Router->executeController()en index.php (línea 23)`

Alguna idea de que puede ser?

**Comment by kriollo** [comment]
efectivamente, subire la nueva base de datos, para que puedas revisar la pagina, saludos.
